### DISCUSSION TITLE
自动连播只连播多P视频而不连播推荐视频

### DISCUSSION THREAD

**Discussion by mirion-dev** [the1812/Bilibili-Evolved#1742] [功能建议 / Ideas]
如题。
b站的自动连播开启后，看完视频就会跳到下一个推荐视频；关闭后，多P视频又无法自动连播下一P。
推荐视频和原视频仅是题材相似而内容具有较大的差异（比如看个高等数学推荐个初高中数学），所以我不希望它连播推荐视频
能否增加一个按钮实现上述功能？
（可以考虑这样实现：默认关闭连播，在播放多P视频时自动打开连播，到最后一P时关闭连播）

**Comment by baka9moe** [comment]
+1，同想要这个功能

**Reply by Geeyun-JY3** [reply]
这功能在 [v1.12.5](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.5-preview) 就已经做了，叫 `传统连播模式`。
认真看正式版 [v1.12.8](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.8) 的 Release Note：
> - 原先存在于预览版的传统连播模式仍有问题, 本次更新不直接开放. 如果要强行开启, 可以在浏览器控制台运行:
> ```
> bilibiliEvolved.settings.autoPlayControl = true
> ```

另外看看这：https://github.com/the1812/Bilibili-Evolved/issues/1710#issuecomment-821759800

近期另外的解决方案可参考：https://github.com/the1812/Bilibili-Evolved/discussions/1634#discussioncomment-593169
